mmpretrain/visualization/visualizer.py

```diff
@@ -167,6 +143,65 @@ def visualize_cls(self,
 
         return drawn_img
 
+    def draw_pred(self,
+                  data_sample: DataSample,
+                  classes: Optional[Sequence[str]],
+                  draw_pred: bool,
+                  draw_score: bool,
+                  texts: Sequence[str],
+                  parent_task: str = ''):
+        if isinstance(data_sample, MultiTaskDataSample):
+            for task in data_sample.tasks:
+                sub_task = f'{parent_task}_{task}' if parent_task else task
+                self.draw_pred(
+                    data_sample.get(task), classes, draw_pred, draw_score,
+                    texts, sub_task)
+        else:
+            if draw_pred and 'pred_label' in data_sample:
+                idx = data_sample.pred_label.tolist()
+                score_labels = [''] * len(idx)
+                class_labels = [''] * len(idx)
+                if draw_score and 'pred_score' in data_sample:
+                    score_labels = [
+                        f', {data_sample.pred_score[i].item():.2f}'
+                        for i in idx
+                    ]
+
+                if classes is not None:
+                    class_labels = [f' ({classes[i]})' for i in idx]
+
+                labels = [
+                    str(idx[i]) + score_labels[i] + class_labels[i]
+                    for i in range(len(idx))
+                ]
+                prefix = f'{parent_task} Prediction: ' if parent_task \
+                    else 'Prediction: '
+                texts.append(prefix + ('\n' + ' ' * len(prefix)).join(labels))
+
+    def draw_gt(self,
+                data_sample: DataSample,
+                classes: Optional[Sequence[str]],
+                draw_gt: bool,
+                texts: Sequence[str],
+                parent_task: str = ''):
```

Comment:
Consider moving these methods as inner functions in the `visualize_cls` method, since they are only for classification tasks.